org.cvfosammmm.Setzer.json

```diff
@@ -0,0 +1,98 @@
+{
+    "app-id": "org.cvfosammmm.Setzer",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.30",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.texlive"],
+    "command": "setzer",
+    "finish-args": [
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--socket=pulseaudio",
+        "--filesystem=host",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--env=PATH=/app/bin:/usr/bin:/app/bin/x86_64-linux",
```

Comment:
Any reason for doing this?

Reply:
I think I just saw those lines somewhere (maybe in the flatpak docs) and put it in. So no specific reason.

Reply:
It's definitely not needed.

Reply:
Which lines exactly should I remove?

Reply:
17th. While on it, `--filesystem=host` should be changed to `home` or `xdg-document`. Users can always expand allowed directories locally.

Reply:
I saw TeXstudio has it set to "host" with a comment "lets LuaLaTeX etc. write output files, even if the document is stored outside of ~" https://github.com/flathub/org.texstudio.TeXstudio/blob/master/org.texstudio.TeXstudio.yaml So I think that is needed in my case as well. They also have "line 17", but without comment. I have removed it, so we'll see if it works out.

Reply:
I tried the last build and it wouldn't find the LaTeX interpreters. So I put the line back in. Ok?